src/Famix-PharoSmalltalk-Tests/FamixStSourcedEntityTest.class.st

```diff
@@ -0,0 +1,50 @@
+"
+I am an abstract test class for the tests of FamixStSourcedEntity's subclasses
+"
+Class {
+	#name : #FamixStSourcedEntityTest,
+	#superclass : #FamixStEntityTest,
+	#category : #'Famix-PharoSmalltalk-Tests'
+}
+
+{ #category : #'as yet unclassified' }
+FamixStSourcedEntityTest >> actualClass [ 
+		^ FamixStSourcedEntity
+]
+
+{ #category : #tests }
+FamixStSourcedEntityTest >> testAddComment [
+	| inst comment |
+	inst := self actualClass new.
+	comment := FamixStComment content: 'This is a test'.
+	inst addComment: comment.
+	self assert: inst comments size equals: 1.
+	self assert: inst comments anyOne equals: comment.
+	self assert: comment container equals: inst
+]
+
+{ #category : #tests }
+FamixStSourcedEntityTest >> testAddCommentFromString [
```

Comment:
This test does not make sense anymore if we cannot att a comment from a string. We should remove it.